### ISSUE TITLE
[WEBSITE]: Update README

### ISSUE THREAD

**Issue by WillTheDeveloper** [wonoly/roadmap#27]
Update the README with some docs such as how to setup etc.

**Comment by mauro-balades** [comment]
btw @WillTheDeveloper. Do the unit tests test if the app can run successfully?

**Comment by WillTheDeveloper** [comment]
I am pretty sure there is a test that does that on a very basic level but I will be writing tests for stuff that we add so don't worry about it. Don't merge a PR if the tests fail!

**Comment by mauro-balades** [comment]
> I am pretty sure there is a test that does that on a very basic level but I will be writing tests for stuff that we add so don't worry about it. Don't merge a PR if the tests fail!

Alright, try to make a test to check if all styles and scripts compile successfully

**Comment by WillTheDeveloper** [comment]
I don't think that is a thing. I'm sure it will be okay. It's not in production yet anyway so I wouldn't worry TOO much.

**Comment by mauro-balades** [comment]
ok

> I don't think that is a thing. I'm sure it will be okay. It's not in production yet anyway so I wouldn't worry TOO much.
